### PR TITLE
Small documentation improvements

### PR DESCRIPTION
**Pull request by bokobza**
Fixed a few typos and made a few bits of text more accurate.

### CODE REVIEW

File: doc/style.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Grin Style Guide
 
-Grin uses [rustfmt](https://github.com/rust-lang-nursery/rustfmt) to maintain consist formatting.
+Grin uses [rustfmt](https://github.com/rust-lang-nursery/rustfmt) to maintain consistant formatting.
```
Comment (antiochp) [inline doc/style.md]: consistent

File: doc/merkle.md [hunk 2]
```diff
@@ -60,7 +58,7 @@ a sum-tree over the commitments of outputs, and the negatives of the commitments
 of inputs.
 
 Input references are hashes of old commitments. It is a consensus rule that
-there are never two identical unspent outputs.
+there never are two identical unspent outputs.
```
Comment (antiochp) [inline doc/merkle.md]: These both feel kind of awkward to me
Maybe this should be reworked as "all unspent outputs must be unique".

File: doc/grin4bitcoiners.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ There are 3 main properties of Grin transactions that make them private:
 2. There are no amounts.
 3. 2 transactions, one spending the other, can be merged in a block to form only one, removing all intermediary information.
 
-The 2 first properties mean that all transactions look the same. Unless you directly participated in the transaction, all inputs and outputs look like random pieces of data (in lingo, they're all random curve points).
+The 2 first properties mean that all transactions look similar. Unless you directly participated in the transaction, all inputs and outputs look like random pieces of data (in lingo, they're all random curve points).
```
Comment (antiochp) [inline doc/grin4bitcoiners.md]: I think this should be `the same`. They are not just similar - they all look exactly like random data.

Comment (bokobza) [reply doc/grin4bitcoiners.md]: That's why I changed to `similar`. They look similar but not `the same`. The same for me means exactly the same.

The Cambridge dictionary says [here](https://dictionary.cambridge.org/grammar/british-grammar/adjectives/same-similar-identical)
> Same means that two or more things are exactly like one another.

> We use similar if two or more things are not entirely the same, or identical if two or more things are exactly the same.

File: doc/merkle.md (hunk 2 above)
Comment (bokobza) [reply doc/merkle.md]: Yeah that's a good idea :-)

File: doc/style.md (hunk 1 above)
Comment (bokobza) [reply doc/style.md]: :grimacing: oh no, sorry.

File: doc/grin4bitcoiners.md (hunk 3 above)
Comment (antiochp) [reply doc/grin4bitcoiners.md]: Not _identical_ in that they are all the same exact piece of random data - but they're not similar to random data, they _are_ random data.

Comment (antiochp) [reply doc/grin4bitcoiners.md]: Maybe we want to say "indistinguishable" here and remove the ambiguity.
But we're getting close to cryptography terminology there that I'll defer to other people for.

Comment (bokobza) [reply doc/grin4bitcoiners.md]: ah! I was just writing that.
I can leave `the same` if you want :-)

Comment (bokobza) [reply doc/grin4bitcoiners.md]: Yeah that's what I understood when I changed `the same` to `similar`.
How about (last try I promise):
`The 2 first properties mean that all transactions are indistinguishable from one another. Unless you directly participated in the transaction, all inputs and outputs look like random pieces of data (in lingo, they're all random curve points).`

Comment (antiochp) [reply doc/grin4bitcoiners.md]: :+1:

**Pull request merged**